tests/load/snowflake/test_snowflake_configuration.py

```diff
@@ -60,6 +60,31 @@ def test_to_connector_params() -> None:
         warehouse='warehouse1',
         role='role1',
     )
+    
+    pkey_str = Path('./tests/common/cases/secrets/encrypted-private-key-base64').read_text('utf8')
```

Comment:
test is not passing because the RSA key string got decoded as base64 into a garbage that could not be decoded as utf-8 :) 

you need to catch decoding exceptions or just all exceptions